proxy/api/src/http/project.rs

```diff
@@ -575,21 +578,48 @@ mod test {
         let owner = ctx.peer_api.init_owner(&ctx.signer, "cloudhead")?;
 
         coco::control::setup_fixtures(&ctx.peer_api, &ctx.signer, &owner)?;
-        let project = &project::list_projects(&ctx.peer_api)?[0];
+
+        let projects = project::Projects::list(&ctx.peer_api)?;
+        let project = projects.into_iter().next().unwrap();
```

Comment:
I don't think this is quite going to work how you expect it to. Because, since our peer created the project we have `signed_refs` in it.

To test this we need to create a second peer we clone the project from and then list our tracked projects, which should consist of the project we cloned.